index.html

```diff
@@ -70,8 +70,12 @@ <h3 class="major">{{post.title}}</h3>
             <h2 class="major">Categories</h2>
 
             <ul class="list-6 categories">
-                {% for cat in site.categories %}
-                <li><a href="/categories/{{cat[0] | downcase | replace:' ','-' }}">{{cat[0]}}</a></li>
+                {% assign shown_categories = '' %}
+                {% for cat in site.categories %} {% assign cat_ = cat[0] | downcase | replace:' ','-' %}
+                {% unless shown_categories contains cat_ %}
```

Comment:
Does this do an exact match on the string values?
When comparing the categories to the production site I've noticed that **Angular** is missing where as **AngularJS-TypeScript** is still there.

[http://localhost:4000/categories/angular/](http://localhost:4000/categories/angular/) is still there though.

Reply:
Hmmm... That might be possible. :)
In the tags we don't have the issue with **Angular** tag at least as it's already displayed before the **Angular XXX** tags are added. But it's possible that other tags might be missing.